solr/Dockerfile

```diff
@@ -21,3 +21,6 @@ WORKDIR /opt/solr
 EXPOSE 8983
 
 COPY rootfs /
+
+RUN mkdir -p /opt/solr/server/solr/ISLANDORA/conf/
+RUN cp /solr_config/* /opt/solr/server/solr/ISLANDORA/conf/
```

Comment:
This actually has no net effect; it places content into `/opt/solr/server/solr/ISLANDORA/conf/` _at image build time_, but this content becomes invisible when a volume is mounted at `/opt/solr`

So, the solution is to copy the files _at runtime_.  To do that, a `cp` statement should be placed into container startup script.  Specifically, this one:  [rootfs/etc/cont-init.d/02-solr-setup.sh](https://github.com/jhu-idc/idc-isle-buildkit/blob/main/solr/rootfs/etc/cont-init.d/02-solr-setup.sh)